omnigan/trainer.py

```diff
@@ -871,6 +872,21 @@ def get_masker_loss(self, multi_domain_batch):  # TODO update docstrings
                         batch_domain
                     ] = update_loss.item()
 
+                    # Then GroundIntersection loss
+                    if self.opts.gen.m.use_ground_intersection:
+                        if self.verbose > 0:
+                            print("Using GroundIntersection loss.")
+                        update_loss = (
+                            self.losses["G"]["tasks"][update_task]["gi"](
+                                prediction, update_target
+                            )
+                            * lambdas.G[update_task]["gi"]
```

Comment:
I think we should only use GI on the real world otherwise this signal will conflict with the actual ground truth on slopes in the simulated world (as you know, ground != physical horizontal)